docs/guides/B2B-Suite/b2b-organizations/bulk-import-spreadsheet.md

```diff
@@ -0,0 +1,97 @@
+# Bulk Import Spreadsheet
```

Comment:
Faltou adicionar o [frontmatter](https://github.com/vtexdocs/dev-portal-content?tab=readme-ov-file#what-information-goes-in-the-frontmatter)